---
authors: Freek Pols
updated: December 3, 2025
---
# Cappucino

## Introductie
Heb je al eens bedacht dat het verwarmen van melk voor een cappucino eigenlijk heel snel gaat zonder dat de melk verdunt wordt?
De melk wordt ook niet echt plaatselijk verhit door een warmte element.
In plaats daarvan wordt er stoom door de melk geblazen.
De stoom condenseert in de melk en geeft daarbij zijn latente warmte af.
Hierdoor warmt de melk snel op zonder dat er (significant veel) water aan toegevoegd wordt.

```{warning}
Stoom is heel heet en kan voor vervelende brandplekken zorgen.
Wees dus voorzichtig bij het uitvoeren van dit practicum.

Daarnaast, nadat de kookplaat uitgaat is er een onderdruk in de kolf.
Hang dan ook de tube uit het water wanneer je de kookplaat uitzet, anders kan de kolf imploderen.
```

In dit practicum gaan we de verdampingswarmte van water bepalen door middel van een zelfgebouwde cappucino machine.
Het principe is als volgt: We gebruiken de opstelling weergegeven in {numref}`Figuur {number} <fig_cappucino>` waarbij we water verwarmen in een kolf met behulp van een kookplaat.
Door de warmte gaat het water koken en ontstaat er stoom.
De stoom stroomt via een tube naar een afgesloten maatbeker met water.
De stoom condenseert in het maatbeker en geeft daarbij zijn latente warmte af.
Door de temperatuurstijging van het water te meten, evenals de hoeveelheid gram water dat verdampt is, kunnen we de verdampingswarmte van water bepalen.

```{figure} figures/verdampingswarmte_setup.png
:width: 70%
:label: fig_cappucino

Een schematische weergave van de cappucino opstelling.
```

## Theorie

De latente verdampingswarmte van water bedraagt 2257 kJ/kg, dit is veel meer dan de specifieke warmtecapaciteit van water (4.18 kJ/kgK).
Wanneer we de waterdamp (g) door koud water leiden, gaan we er van uit dat de waterdamp condenseert en daarbij zijn latente warmte afgeeft aan het koude water.
Door te bepalen hoeveel gram water verdampt is en hoeveel de temperatuur van het koude water stijgt, kunnen we de latente warmte van verdamping bepalen:

$$ Q_{condensatie} = m_{damp} L = m_{water} c \Delta T $$

met $m_{damp} = \Delta m_{kolf}$.

## Methoden en materialen

### Materialen
- Warmteplaat
- Kolf met stop en tube
- Maatcilinder
- Thermometer
- Weegschaal
- Water


### Procedure

- Vul de kolf met ongeveer 100 mL water: bepaal precies de massa water ($m_{w_1}$).  
- Vul de maatcilinder met ongeveer 100 mL water: bepaal precies de massa water ($m_{bad}$).
- Bepaal de temperatuur van dit waterbad ($T_{bad_1}$).
- Zet de kolf op de warmteplaat en zet de warmteplaat aan - maximale stand 3, zorg ervoor dat de tube goed in het waterbad hangt.
- Wacht tot de temperatuur van het waterbad met ongeveer 20 K is gestegen.
- Stop de meting / zet de warmteplaat uit. Noteer meteen de temperatuur($T_{bad_2}$) en haal de tube uit het water! 

```{warning}
Pas op stoom is heet!
```

- Bepaal precies de massa water in de kolf ($m_{w_2}$).
- Verwerk je resultaten hieronder om de latente warmte van verdamping van water te bepalen.

## Resultaten

In [ ]:
# Partner bij dit practicum: Florian Spauwen

import numpy as np
#resultaten_1

# Toestand 1
m_w_1 = 0.048344       # kg
m_bad_1 = 0.117587 - 0.069286        # kg
T_bad_1 = 20.25    + 273.15     # K

# Toestand 2
m_w_2 = 0.0426      # kg
T_bad_2 = 40.5 + 273.15     # K
m_bad_2 = 0.120089 - 0.069286

c = 4186 #J/kg K

mdamp = m_w_1-m_w_2
deltaT = T_bad_2-T_bad_1

#Bepaling van de latente verwarming
L = (m_bad_1*c*deltaT) / mdamp
print('De latente verwarming van verdamping van water is', np.round(L / 1000, 0), 'kJ/kg')

De latente verwarming van verdamping van water is 713.0 kJ/kg


In [30]:
# De eerste metingen geven een te laag resultaat, dus er is nog een keer gemeten. Hieronder de tweede metingen.

# Data toestand 1
m_w_3 = 0.049046 # kg
m_bad_3 = 0.117602 - 0.069286 #kg
T_bad_3 = 17.75 + 273.15 # K

# Data toestand 2
m_w_4 = 0.044 #kg
T_bad_4 = 43+273.15 #K
m_bad_4 = 0.0513 #kg

# Functie om de latente verwarming te bepalen
def latent(deltaT):
    return((((m_bad_3)*c*(deltaT))/(m_w_3-m_w_4)))

deltaTnieuw = T_bad_4-T_bad_3
print('De latente verwarming van verdamping van water is', np.round(latent(deltaTnieuw)/1000, 0), 'kJ/kg')


De latente verwarming van verdamping van water is 1012.0 kJ/kg


## Discussie en conclusie

# Discussie

Bij de tweede keer meten wordt een latente verdampingswarmte bepaald van 1012 kJ/kg. Dit is minder dan de helft van de literatuurwaarde van 2257 kJ/kg. De oorzaak van deze grote fout in de waarde is hoogstwaarschijnlijk dat er tijdens het meten van de temperatuur niet geroerd werd in het water, waardoor de temperatuur niet homogeen werd gemeten. Bij een proef als dit, waarbij er een zeer kleine verandering is in de massa van het water (slechts 3 gram bij een temperatuursverandering van 20 K), maakt een klein verschil in de positie van de thermometer al snel een groot verschil. De thermometer werd vrij laag in het waterbad gehouden en er werd niet geroerd, waardoor wanneer de warmte stijgt, er een te lage temperatuur wordt gemeten. Hierdoor is delta T lager gemeten dan het echt is, en komt er dus een te lage latente verdampingswarmte uit.

# Conclusie

Dit experiment is uitgevoerd met het doel om de latente verdampingswarmte van water te bepalen. Een kolf en een maatcilinder zijn beiden gevuld met water, de kolf met 49 gram water, en de maatcilinder met 48.3 gram water. De kolf is op een warmteplaat gezet en opgewarmd, terwijl een buis warmte overdroeg van de kolf naar de maatcilinder. Bij een temperatuursstijging van 20 K is de temperatuur van de maatcilinder en de massa's van beiden opnieuw gemeten. Uit de data analyse volgt dat de latente verdampingswarmte van water gelijk is aan 1012 kJ/kg. Dit ligt ver van de literatuurwaarde van 2257 kJ/kg. De oorzaak van de fout is dat bij het meten van de temperatuur, de temperatuur niet homogeen werd gehouden terwijl de warmte stijgt, en er een lager verschil in temperatuur is gemeten in het waterbad dan er echt was. In een vervolgonderzoek kan dit voorkomen worden door te roeren in het waterbad zodat de temperatuur overal zo gelijk mogelijk is, en door de thermometer precies in het midden van het waterbad te houden.